### ISSUE TITLE
Train and Dataset issue

### ISSUE THREAD

**Issue by Paranoidv** [ansleliu/LightNet#5]
May I ask how to train model with both gtFine and gtCoarse dataset? And another question, how to get the model mIOU when test on val dataset?Thank you !

**Comment by ansleliu** [comment]
@Paranoidv If you want to train the LightNet with both "gtFine" and "gtCoarse", please merge them together when using my dataloader or write you a new dataloader for this situation. ^_^

**Comment by ansleliu** [comment]
@Paranoidv 

> how to get the model mIOU when test on val dataset? 
Please run "evaluation_cityscapes.py" for the model you want to test (for cityscape test or val set) to create masks, and then evaluate those masks using "evalPixelLevelSemanticLabeling.py". ^_^

**Comment by Paranoidv** [comment]
Thank you for your kindly answer, May I ask how much mIOU when you train SE-WResNetV2 without pretrained wide-resnet ? because I gei low mIOU when I train SE-WResNetV2 from scratch, about mIOU=0.50?

**Comment by Paranoidv** [comment]
@ansleliu 
Thank you for your kindly answer, May I ask how much mIOU when you train SE-WResNetV2 without pretrained wide-resnet ? because I gei low mIOU when I train SE-WResNetV2 from scratch, about mIOU=0.50?

**Comment by Paranoidv** [comment]
@ansleliu 
Thank you for your kindly answer, May I ask how much mIOU when you train SE-WResNetV2 without pretrained wide-resnet ? because I gei low mIOU when I train SE-WResNetV2 from scratch, about mIOU=0.50

**Comment by fip-lab** [comment]
> May I ask how to train model with both gtFine and gtCoarse dataset? And another question, how to get the model mIOU when test on val dataset?Thank you !

hi, Which code to use to run the gtFine data set? Thank you very much!
